docs/20_tutorial.md

```diff
@@ -14,10 +15,161 @@ the computers read those ballots, to determine whether the computers
 did their job accurately…” Try to put their minds at ease about the
 process. :)
 
+Terms and process for the RLA are defined and prescribed in Rule 25.
+*** citation for Rule 25.
+
 ***should the run book cover the mechanics of storing ballots and the
 other physical aspects of the RLA process which are not ‘in’ the RLA
 tool/system?
 
+### What the RLA Tool Does Not Do
+
+Although the RLA process requires Counties to 
+generate and preserve
+a Summary Results Report and a Results File Export, the RLA Tool 
+does not interact with these files in any way. Nor does the RLA Tool 
+aid in any way with the CVR Export Verification required by the 
+RLA process. 
+
+While the RLA Tool does provide for upload of the 
+hashed Ballot Manifest, the hashed CVR file and their hashes, 
+the RLA Tool does not provide a hashing utility.
+***review please!
+
+The RLA Tool does not provide any support for paper ballot retrieval or 
```

Comment:
This isn't entirely true.  On the audit board dashboard we can/will display the ballot manifest information for the selected ballots under audit, including scanner id, batch id, and the storage container that should contain the ballot in question.  I'm right about this, @dmzimmerman @MorganMillerUX @nealmcb, aren't I?